### PR TITLE
fix: upgrade btn bug

### LINKED ISSUE #9102: [CAL-1763] Download button disappeared
Found a bug? Please fill out the sections below. 👍

Reported here https://github.com/calcom/cal.com/pull/6411#issuecomment-1561859521

<sub>[CAL-1763](https://linear.app/calcom/issue/CAL-1763/download-button-disappeared)</sub>

### PR DESCRIPTION
**Pull request by Udit-takkar**
## What does this PR do?

Fixes https://github.com/calcom/cal.com/issues/9102

Download button is working fine and only visible when user has teams plan. the upgrade button should render in case user is not part of any active team which was not happening https://calendso.slack.com/archives/C01NXDSRY9W/p1684959944326289?thread_ts=1684958912.389019&cid=C01NXDSRY9W

 
Screenshot 2023-05-25 at 2 19 53 AM

**Environment**: Staging(main branch) / Production

## Type of change

- Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: packages/features/ee/video/components/UpgradeRecordingBanner.tsx [hunk 1]
```diff
@@ -1,31 +0,0 @@
-import { useRouter } from "next/router";
```
Comment (Udit-takkar) [inline packages/features/ee/video/components/UpgradeRecordingBanner.tsx:1]: This file is not used anywhere

File: packages/features/ee/video/ViewRecordingsDialog.tsx [hunk 2]
```diff
@@ -138,7 +133,12 @@ const ViewRecordingsList = ({ roomName, hasTeamPlan }: { roomName: string; hasTe
                     {t("download")}
                   </Button>
                 ) : (
-                  <UpgradeTeamsBadge />
+                  <Button
+                    tooltip={t("upgrade_to_access_recordings_description")}
```
Comment (Udit-takkar) [inline packages/features/ee/video/ViewRecordingsDialog.tsx:137]: added this tooltip message

**Comment by PeerRich** [comment]
can you also fix the border? its not supposed to be `border` only I think you need to add `border-subtle`

**Comment by Udit-takkar** [comment]
Screenshot 2023-05-25 at 3 10 37 PM

**Review by sean-brydon — APPROVED** [review]

**Review by sean-brydon — APPROVED** [review]
[birdbox](https://app.graphite.dev/macros/calcom)

**Review by zomars — APPROVED** [review]

**Pull request merged**